aws/fedora-36-aarch64/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "user": "fedora",
+    "runnerArch": "aarch64"
```

Comment:
Shouldn't this be `arm64`?

Reply:
I don't know, the other arm definitions have `aarch64` too... And I don't know what's wrong with the `x86_64` one :/